charts/rancher-istio/1.1.5/charts/prometheus/templates/deployment.yaml

```diff
@@ -68,6 +67,31 @@ spec:
             mountPath: /etc/prometheus
           - mountPath: /etc/istio-certs
             name: istio-certs
+        - name: prometheus-nginx
+          command:
+          - /bin/sh
+          - -c
+          - cp /nginx/run-sh.tmpl /var/run/nginx-start.sh; chmod +x /var/run/nginx-start.sh; /var/run/nginx-start.sh
```

Comment:
From what I see, we are not using `run-sh.tmpl` as a template file, can we simply mount a `nginx-start.sh` file to the container and gives it executable permission. So that we don't need to do copy and chmod here, but simply use `/var/run/nginx-start.sh` as the command.

Reply:
I think we should keep this style until we tested the shell script mount from configmap can run successfully.